app/src/main/res/layout/comment_card.xml

```diff
@@ -168,19 +168,30 @@
 
     <androidx.appcompat.widget.AppCompatImageView
         android:id="@+id/info_button"
-        style="@style/CommentsReplyButton"
+        style="@style/CommentsInfoIcon"
         android:src="@drawable/ic_info"
-        app:layout_constraintTop_toTopOf="@+id/flagged_message"
-        app:layout_constraintStart_toStartOf="@id/avatar"
-        />
+        app:layout_constraintTop_toBottomOf="@+id/avatar"
+        app:layout_constraintStart_toStartOf="@id/avatar" />
 
     <androidx.appcompat.widget.AppCompatTextView
-        android:id="@+id/flagged_message"
+        android:id="@+id/removed_message"
         style="@style/CommentCardFlaggedMessage"
+        android:visibility="gone"
         android:text="@string/This_comment_has_been_removed_by_Kickstarter"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toEndOf="@id/info_button"
-        app:layout_constraintTop_toBottomOf="@id/avatar"/>
+        app:layout_constraintTop_toBottomOf="@id/avatar"
+        tools:visibility="visible"/>
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/flagged_message"
+        style="@style/CommentCardFlaggedMessage"
+        android:visibility="gone"
+        android:text="@string/this_comment_is_under_review"
```

Comment:
Question here @leighdouglas, is there any ticket prepared for translations?

Reply:
Good point! I'll create that 👍🏽